fs/layer/node.go

```diff
@@ -244,6 +244,26 @@ func (fs *fs) inodeOfStatFile() uint64 {
 	return (uint64(fs.baseInode) << 32) | 2 // reserved
 }
 
+// basicTelemetry wraps informational FUSE telemetry operations.
+func (fs *fs) basicTelemetry(ctx context.Context, operationName string, path string) {
+	if fs.logFSOperations {
+		log.G(ctx).WithFields(logrus.Fields{
+			"operation": operationName,
+			"path":      path,
+		}).Debug("FUSE operation")
+	}
+	if fs.operationCounter != nil {
+		fs.operationCounter.Inc(operationName)
+	}
+}
+
+// reportTelemetry wraps FUSE failure telemetry operations.
+func (fs *fs) reportTelemetry(operationName string, stateError error) {
```

Comment:
nit: name does not communicate to me the failure state requirement for this function.